cpp/modmesh/toggle/RadixTree.hpp

```diff
@@ -179,5 +200,151 @@ class RadixTree
     key_type m_unique_id = 0;
 }; /* end class RadixTree */
 
+// The profiling result of the caller
+struct CallerProfile
+{
+    CallerProfile()
+        : total_time(0)
```

Comment:
Why don't you give it a default value and avoid this custom default constructor?  By the rule of 5, if you customize one of the 5, you should explicitly spell out all of the 5.

Reply:
unfortunately, I found that `std::chrono::nanoseconds` needs to explicitly assign `0` to have the correct behavior.
sure, I will add the other 4.

Reply:
I still set them as default, not sure if it is what you want

Reply:
I meant to use the default value of a member datum:

```cpp
std::chrono::nanoseconds total_time = 0; /// use nanoseconds to have higher precision
```

You already had the following.  I do not see why `total_time` does not use a default value.

```cpp
int call_count = 0;
bool is_running = false;
```

If you use the default value of the member datum, you do not need to spell out any of the constructors.

Reply:
`std::chrono::nanoseconds total_time = 0` is invalid, so I use `std::chrono::nanoseconds total_time = std::chrono::nanoseconds(0);`